app/controllers/events.server.controller.js

```diff
@@ -0,0 +1,107 @@
+const Event = require('mongoose').model('Event');
+
+function getErrorMessage(err) {
+  let message = '';
+
+  if (err.code) {
+    switch (err.code) {
+        case 11000:
+        case 11001: 
+          message = 'Email already exists';
+          break;
+        default: 
+          message = 'Something went wrong';
+     }
+   } else {
+     for (var errName in err.errors) {
+       if (err.errors[errName].message){
+         message = err.errors[errName].message;
+       }
+     }
+   }
+   return message;
+};
+
+exports.create = function(req, res){
```

Comment:
Consider changing to createEvent for consistency with view and route calling, add space between ) and {